Project2015To2017/Analysis/Diagnostics/W021SystemNuGetPackagesDiagnostic.cs

```diff
@@ -0,0 +1,223 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Project2015To2017.Definition;
+
+namespace Project2015To2017.Analysis.Diagnostics
+{
+	public class W021SystemNuGetPackagesDiagnostic : DiagnosticBase
+	{
+		private const string MaxTargetFramework = "net499";
+		private const string MaxTargetStandard = "netstandard9.9";
+
+		public W021SystemNuGetPackagesDiagnostic() : base(21)
+		{
+		}
+
+		private static readonly string[] IncompatiblePrefixes = {"net1", "net2", "net3", "net40"};
+
+		private static readonly string[] Packages45 =
+		{
+			"System.Collections",
```

Comment:
This array and 2 more below are generated automatically by script. The source is ReferenceAssemblies\4.X\System.*.dll and ReferenceAssemblies\4.7.2\Facades\*.dll. Verified against NuGet with same name and with mandatory .NETFramework/.NETStandard TFMs.